tests/client/amo/components/TestFeaturedAddons.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { findDOMNode } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import * as featuredActions from 'amo/actions/featured';
+import {
+  FeaturedAddonsBase,
+  mapStateToProps,
+} from 'amo/components/FeaturedAddons';
+import createStore from 'amo/store';
+import I18nProvider from 'core/i18n/Provider';
+import { fakeAddon } from 'tests/client/amo/helpers';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+describe('<FeaturedAddons />', () => {
+  const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
+  function render({ ...props }) {
+    return findDOMNode(findRenderedComponentWithType(renderIntoDocument(
+      <Provider store={createStore(initialState)}>
+        <I18nProvider i18n={getFakeI18nInst()}>
+          <FeaturedAddonsBase i18n={getFakeI18nInst()} {...props} />
```

Comment:
Actually using `shallowRender` here might be doable and would clean this up, as removing the `i18n` prop here breaks stuff but including `FeaturedAddons` instead of `FeaturedAddonsBase` means passing stuff into the `<Provider>` from each test and is gross.

I'll try to shallowrender it if it's easy.